packages/zapp/console/src/components/common/ReactJsonView.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import ReactJsonView, { ReactJsonViewProps } from 'react-json-view';
+import * as copyToClipboard from 'copy-to-clipboard';
+import { primaryTextColor, smallFontSize } from 'components/Theme/constants';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  jsonViewer: {
+    marginLeft: '-10px',
```

Comment:
Per Old to new comparison - loos like we would need to make font size a bit bigger for jsonViewer, if it's possible:
<img width="427" alt="Screen Shot 2022-05-04 at 10 25 46 AM" src="https://user-images.githubusercontent.com/55718143/166809359-83ca89ec-6c70-49da-ac79-f762353fbb1c.png">